lib/install.js

```diff
@@ -248,6 +248,8 @@ function install(gyp, argv, callback) {
     opts.ca = gyp.opts.ca;
     opts.cafile = gyp.opts.cafile;
 
+    // example
+    // "https://mapbox-node-pre-gyp-public-testing-bucket.s3.us-east-1.amazonaws.com/node-pre-gyp/node-pre-gyp-test-app1/v0.1.0/Release/node-v72-linux-x64.tar.gz"
```

Comment:
Recommend removing this comment since the actual path is going to differ depending on the package.json `binary` details.

Reply:
will do.